### PR TITLE
PaymentResponse.complete() method

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PASSED

Started: 2017-08-21 05:24:15
Finished: 2017-08-21 05:28:57



View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/17583)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/266680974)

**Comment by w3c-bots** [comment]
# Build PASSED

Started: 2017-08-21 05:24:15
Finished: 2017-08-21 05:28:57



View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/17583)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/266680974)

File: payment-request/payment-response/helpers.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
+/**
+ * Pops up a payment sheet, allowing options to be
+ * passed in if particular values are needed.
+ *
+ * @param PaymentOptions options
+ */
+setup({ explicit_done: true, explicit_timeout: true });
```
Comment (marcoscaceres) [inline payment-request/payment-response/helpers.js]: moving this up to the top of the file

**Review by edenchuang — APPROVED** [review]
LGTM

**Pull request merged**